core/tauri/src/endpoints/notification.rs

```diff
@@ -68,80 +60,23 @@ impl Cmd {
     Ok(())
   }
 
-  #[cfg(notification_all)]
-  fn request_notification_permission<R: Runtime>(
-    context: InvokeContext<R>,
-  ) -> super::Result<&'static str> {
-    if request_permission(&context) {
-      Ok(PERMISSION_GRANTED)
-    } else {
-      Ok(PERMISSION_DENIED)
-    }
-  }
-
-  #[cfg(not(notification_all))]
   fn request_notification_permission<R: Runtime>(
     _context: InvokeContext<R>,
   ) -> super::Result<&'static str> {
-    Ok(PERMISSION_DENIED)
-  }
-
-  #[cfg(notification_all)]
-  fn is_notification_permission_granted<R: Runtime>(
-    context: InvokeContext<R>,
-  ) -> super::Result<Option<bool>> {
```

Comment:
This may not be the appropriate time to address it, but is there a reason this isn't just `Result<bool>`?

Reply:
changed it

Reply:
it previously returned `None` when the permission was never asked, so like an unknown state

Reply:
Maybe I'm just looking at the wrong place in this file. But if it's changed in rust then would be good to change over in TS as well